gnomekeyring_p.h

```diff
@@ -45,7 +45,7 @@ class GnomeKeyring : private QLibrary {
         } attributes[32];
     } PasswordSchema;
 
-    typedef void ( *OperationGetStringCallback )( Result result, bool binary,
+    typedef void ( *OperationGetStringCallback )( Result result,
```

Comment:
I wonder if there was an API change in the Gnome keyring that might make this change incompatible with older versions?

Reply:
Not sure. 
I also tried the other way around, i.e. adding the `binary` parameter to `gnomeKeyring_readCb` but that broke the execution at runtime with a nice segfault. :disappointed: 
Unfortunately, without fixing this, it is impossible to compile for me due to the signature mismatch warning in function pointer cast. E.g. I got:

>[...]/qtkeychain/keychain_unix.cpp:193: error: cast between incompatible function types from ‘void (*)(int, const char*, QKeychain::JobPrivate*)’ to ‘GnomeKeyring::OperationGetStringCallback’ {aka ‘void (*)(GnomeKeyring::Result, bool, const char*, void*)’} [-Werror=cast-function-type]
                                                    reinterpret_cast<GnomeKeyring::OperationGetStringCallback>( &JobPrivate::gnomeKeyring_readCb ),
                                                                                                                                                 ^